src/main/java/uk/gov/hmcts/reform/sandl.snlevents/repository/db/HearingPartRepository.java

```diff
@@ -13,4 +15,41 @@
 public interface HearingPartRepository extends JpaRepository<HearingPart, UUID> {
 
     List<HearingPart> findBySessionIn(Collection<Session> sessions);
+
+    @Query(nativeQuery = true,
+        value = "select cast(title as varchar(100)) as title, cast(hearings as int) as hearings, \n"
```

Comment:
can this query string be loaded from file? + it would be good if you can add some comment to it, explaining what it does

Reply:
sl 976